docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -661,6 +661,169 @@ public class PersonRepository implements PanacheRepositoryBase<Person,Integer> {
 }
 ----
 
+== Mocking
+
+=== Using the “entity pattern”
+
+If you are using the “entity pattern” you cannot use Mockito directly as it does not support mocking static methods,
+but you can use the `quarkus-panache-mock` module which allows you to use Mockito to mock all provided static
+methods, including your own.
+
+Given this simple entity:
+
+[source,java]
+----
+@Entity
+public class Person extends PanacheEntity {
+
+    public String name;
+
+    public static List<Person> findOrdered() {
+        return find("ORDER BY name").list();
+    }
+}
+----
+
+You can write your mocking test like this:
+
+[source,java]
+----
+@QuarkusTest
+public class PanacheFunctionalityTest {
+
+    @Test
```

Comment:
This one is not disabled on native but the repository variant is ?
As injection didn't works in native mode you should add a note that explain it

Reply:
this is not a native test, I've removed that annotation